lib/start.js

```diff
@@ -30,7 +30,12 @@ const start = (buildConfig = config.defaultBuildConfig, options) => {
   }
 
   if (process.platform === 'darwin') {
-    util.run(path.join(config.outputDir, 'Brave.app', 'Contents', 'MacOS', 'Brave'), braveArgs, cmdOptions)
+    let product_name = 'Brave-Browser'
```

Comment:
this is fine to merge, but I think it probably makes sense to move this logic to config.js

Reply:
Yep. agreed. Will do refactoring in the follow-up pr.